front/app/containers/Authentication/messages.ts

```diff
@@ -21,9 +21,10 @@ export default defineMessages({
     id: 'app.containers.SignUp.signUp2',
     defaultMessage: 'Sign up',
   },
-  invitationError: {
-    id: 'app.containers.SignUp.invitationError',
-    defaultMessage: 'Your invitation has expired or has already been redeemed.',
+  invitationErrorMessage: {
+    id: 'app.containers.SignUp.invitationErrorMessage',
+    defaultMessage:
+      'Your invitation has expired or has already been used. Please sign up to continue.',
```

Comment:
Hmmm, not super clear what this means. "Invitation has been used; sign up to continue". Can we clarify this some more? What does "an invitation has been used" mean here? That the email has been taken by someone else?

Reply:
You are already using a much clearer explanation. So basically:
- Invite expired => sign up
- Invite used => sign in

> Your invitation has expired or has already been used. If you have already used the invitation link to create an account, try signing in. Otherwise, sign up to create a new account.

Something like this? Would be better if we can distinguish between the two and show a shorter, more focused error message.